contracts/pods/PodManager.sol

```diff
@@ -0,0 +1,136 @@
+/// can use to deploy more optimistic governance pods. It will create an
+/// Orca pod and deploy an optimistic timelock alongside it.
+///
+/// The timelock and Orca pod are then linked up so that the Orca pod is
+/// the only proposer and executor.
+contract PodManager {
+    /// @notice Address from which the admin pod transactions are sent. Likely a timelock
+    address private immutable podAdmin;
+
+    /// @notice Orca controller for Pod
+    IControllerV1 private immutable podController;
+
+    /// @notice Orca membership token for the pod
+    IMemberToken private immutable memberToken;
+
+    /// @notice Fei core address
+    address private immutable core;
+
+    /// @notice Mapping between podId and it's optimistic timelock
+    mapping(uint256 => address) public getPodTimelock;
+
+    event CreatePod(uint256 podId, address safeAddress);
+    event CreateOptimisticTimelock(address timelock);
+
+    /// @notice Restrict function calls to podAdmin
+    modifier onlyPodAdmin() {
+        require(msg.sender == podAdmin, "Only PodAdmin can deploy");
+        _;
+    }
+
+    constructor(
+        address _core,
+        address _podAdmin,
+        address _podController,
+        address _memberToken
+    ) {
+        require(_core != address(0), "Zero address");
+        require(_podAdmin != address(0x0), "Zero address");
+        require(_podController != address(0x0), "Zero address");
+
+        core = _core;
+        podAdmin = _podAdmin;
+        podController = IControllerV1(_podController);
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    ///////////////////// GETTERS ///////////////////////
+
+    /// @notice Get the address of the Gnosis safe that represents a pod
+    /// @param podId Unique id for the orca pod
+    function getPodSafe(uint256 podId) external view returns (address) {
+        return podController.podIdToSafe(podId);
+    }
+
+    //////////////////// STATE-CHANGING API ////////////////////
+
+    /// @notice Create a child Orca pod. Callable by the DAO and the Tribal Council
+    /// @param _members List of members to be added to the pod
+    /// @param _threshold Number of members that need to approve a transaction on the Gnosis safe
+    /// @param _podLabel Metadata, Human readable label for the pod
+    /// @param _ensString Metadata, ENS name of the pod
+    /// @param _imageUrl Metadata, URL to a image to represent the pod in frontends
+    /// @param minDelay Delay on the timelock
+    function createChildOptimisticPod(
+        address[] memory _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string memory _ensString,
+        string memory _imageUrl,
+        uint256 minDelay
+    ) public onlyPodAdmin returns (uint256, address) {
+        uint256 podId = memberToken.getNextAvailablePodId();
+
+        podController.createPod(
+            _members,
+            _threshold,
+            podAdmin,
```

Comment:
should the podAdmin be a contract rather than an EOA? Then we don't need to choose between the DAO and Tribal Council and it can allow more of a heirarchy

Reply:
I think the podAdmin will likely be a timelock per the comment on it.

Reply:
Currently it's setup so that each admin pod (i.e. Tribe DAO and Tribal Council) will deploy it's own `PodFactory.sol`, and then use that to create it's child pods. So the `podAdmin` is a contract (the timelock of the DAO or Tribal Council).

I thought that was the simplest/quickest way to get started + the `podAdmin` is what has the powers to add, remove signers etc. 

Will have a think about some kind of orchestrator contract that could create the hierarchy, rather than pushing that to the deploy scripts to create

Reply:
As discussed, leaving as is for this MVP